index.md

```diff
@@ -15,6 +15,10 @@ title: Home
 * [LEAN meets MaRDI and OSCAR (5 - 6 Dec, 2024 at TU Berlin).](https://polymake.org/doku.php/workshops/lean_workshop1224)
 * [Computational Algebraic Geometry Workshop (18 - 22 Nov, 2024 at Durham University).](https://sites.google.com/view/durhamcompalggeom/home)
 
+{% for e in site.data.events %}
+  
+{% endfor %}
```

Comment:
?

Reply:
Whoops, this should not be here. (Started writing some code for trying to automate the upcoming events, but decided to look at the low hanging fruit of of issue 277 first)